src/transformers/models/blip_2/modeling_blip_2.py

```diff
@@ -1238,8 +1238,28 @@ def __init__(self, config: Blip2Config):
         # Initialize weights and apply final processing
         self.post_init()
 
-    def get_input_embeddings(self) -> nn.Module:
-        return self.vision_model.embeddings.patch_embedding
+    def get_input_embeddings(self):
```

Comment:
@younesbelkada As seen offline, but could you confirm it's OK ?

It's copy pasted from `Blip2ForConditionalGeneration`. Without those, the tie_weights seems broken after load (This isn't safetensors specific and could be a separate PR)